<?xml version="1.0"?>
<launch>

	<node name="simple_navigation_node" pkg="moverobotic_simple_navigation" type="moverobotic_simple_navigation" output="screen" respawn="true">
		<param name="base_frame_id" value="base_link"/>
		<param name="map_frame_id" value="map"/>
		<param name="home_coordinate" value="1.904753 3.045843 0.000000 0.000000 0.000000 -0.005821 0.999983"/>
		<param name="home_button" value="2"/>
	</node>
	<!-- <node name="test_dynamic_load" pkg="dynamic_reconfigure" type="dynparam" args="load /test_dynamic_node $(find muje_playground)/cfg/test_dynamic.yaml" /> -->
	<node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" required="true"/>

</launch>